packages/php-wasm/universal/src/lib/php-response.ts

```diff
@@ -219,6 +228,33 @@ async function streamToText(
 	}
 }
 
+async function streamToBytes(
+	stream: ReadableStream<Uint8Array>
+): Promise<Uint8Array> {
+	const reader = stream.getReader();
+	const chunks: Uint8Array[] = [];
+
+	while (true) {
+		const { done, value } = await reader.read();
+		if (done) {
+			const totalLength = chunks.reduce(
+				(acc, chunk) => acc + chunk.byteLength,
+				0
+			);
+			const result = new Uint8Array(totalLength);
+			let offset = 0;
+			for (const chunk of chunks) {
+				result.set(chunk, offset);
+				offset += chunk.byteLength;
+			}
+			return result;
+		}
+		if (value) {
+			chunks.push(value);
+		}
+	}
+}
+
```

Comment:
I started to look at this PR yesterday afternoon, but then had to go. However, I thought there was a one-liner to do this, or at least a simple way, and now I found it:

```js
const bytes = await new Response(stream).bytes();
```

It might be worth considering for the simplicity of that code.

Reply:
> ```js
> const bytes = await new Response(stream).bytes();
> ```

Nice and thank you, I hadn't considered that!